docs/install/iam_policy.json

```diff
@@ -0,0 +1,226 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "iam:CreateServiceLinkedRole",
+                "ec2:DescribeAccountAttributes",
+                "ec2:DescribeAddresses",
+                "ec2:DescribeInternetGateways",
```

Comment:
Do we need IGW Describe ?

Reply:
> Do we have write up on how to further scope down when vpcID / clusterName are provided ?

no, it's a TODO item.

Reply:
> Do we need IGW Describe ?

It's need by ELB implicitly during CreateLoadBalancer call: https://docs.aws.amazon.com/elasticloadbalancing/latest/userguide/elb-api-permissions.html

Reply:
LGTM